index.html

```diff
@@ -0,0 +1,880 @@
+            <p>
+              march 10, 2020 — Whether you want to count down to an event,
+              visualize a fundraising campaign, show a clock or a sales counter,
+              Rik Schennink’s open-source flip counter plugin has got you
+              covered.
+            </p>
+          </div>
+        </div>
+        <div class="persons">
+          <div>
+            <img
+              src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/fcff551c-c0dc-42de-af3e-68a1f0161130/braid-opt.png"
+              alt="Braid"
+              class="bio-image-image"
+            />
+          </div>
+          <div class="persons-text">
+            <span>
+              Braid wrote
+            </span>
+            <b>
+              Vue Formulate
+            </b>
+            <p>
+              march 9, 2020 — Forms are often rather tedious to author. Vue
+              Formulate provides a powerful and flexible API that helps
+              developers handling forms with Vue.
+            </p>
+          </div>
+        </div>
+        <div class="persons">
+          <div>
+            <img
+              src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/284ba87d-8b4e-4551-a9ef-631278a46b45/mark-boulton-100px-opt.png"
+              alt="Mark Boulton"
+              class="bio-image-image"
+            />
+          </div>
+          <div class="persons-text">
+            <span>
+              Mark Boulton wrote
+            </span>
+            <b>
+              Leading A Remote Design Team
+            </b>
+            <p>
+              march 6, 2020 — Leading a remote design team can be a bit daunting
+              at first. Mark Boulton shares some simple principles that help you
+              get into the swing of things quickly, so that you can start to
+              become more productive and effective.
+            </p>
+          </div>
+        </div>
+        <div class="persons">
+          <div>
+            <img
+              src="https://cloud.netlifyusercontent.com/assets/344dbf88-fdf9-42bb-adb4-46f01eedd629/d3894a4d-ac15-47a8-b62d-362f1be9eba0/jason-pamental-100px-opt.png"
+              alt="Jason Pamental"
+              class="bio-image-image"
+            />
+          </div>
+
+          <div class="persons-text">
+            <span>
+              Jason Pamental wrote
+            </span>
+            <b>
+              A Variable Fonts Primer
+            </b>
+            <p>
+              march 5, 2020 — With variable fonts, we can accomplish what used
+              to require several files with a single file and some CSS. The “A
+              Variable Fonts Primer” teaches you how variable fonts tick and how
+              they can benefit UI design, accessibility, and long-form reading.
+            </p>
+          </div>
+        </div>
+      </article>
+      <div class="more-info">
+        More links in twitter
+      </div>
```

Comment:
Please the content of this section and use the grayscale design.

Include divs for the images and change the bg color depending on its hierarchy.